### PR TITLE
Upgrade dependences

### PR DESCRIPTION
**Pull request by dongm2ez**

### CODE REVIEW

File: lib/Monitor.php [hunk 1]
```diff
@@ -188,7 +188,7 @@ private function cleanParams($params)
 
     private function cleanMetrics($metrics)
     {
-        return array_map(function ($key) {
+        return array_map(function ($key) use ($metrics) {
```
Comment (aflanagan) [inline lib/Monitor.php:191]: @dongm2ez thanks for catching this! Would you be able to add a test for `cleanMetrics`?

**Comment by aflanagan** [comment]
@dongm2ez thanks for the PR! We'll get a new release out this week that includes this update.

File: lib/Monitor.php (hunk 1 above)
Comment (dongm2ez) [reply lib/Monitor.php:191]: Sure! today or tomorrow.

**Comment by aflanagan** [comment]
thanks @dongm2ez!

**Pull request merged**